k8s_resources/crd.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: envinjectors.envinjector.org
+spec:
+  scope: Namespaced
+  group: envinjector.org
```

Comment:
Lets change it to itoutposts.com. As in a future we will have some additional operators